src/sentry/tasks/deletion.py

```diff
@@ -84,6 +82,18 @@ def run_deletion(deletion_id, first_pass=True):
         transaction_id=deletion.guid,
         actor_id=deletion.actor_id,
     )
+    if not task.should_proceed(instance):
+        logger.info(
+            "object.delete.aborted",
+            extra={
+                "object_id": deletion.object_id,
+                "transaction_id": deletion.guid,
+                "model": deletion.model_name,
+            },
+        )
+        deletion.delete()
```

Comment:
No point in having deletion jobs with `aborted=true` in them when we can just remove those records.